README.md

```diff
@@ -63,6 +63,24 @@ You can find more in-depth examples in the [tests](https://github.com/haydnv/tin
 
 To use Tinychain's **Tensor** feature, which enables automatic GPU acceleration, you'll have to first install [ArrayFire](https://arrayfire.org/docs/installing.htm). Then, follow the instructions for the [ArrayFire Rust wrapper](https://github.com/arrayfire/arrayfire-rust#use-from-cratesio--) to set the `LD_LIBRARY_PATH` and `AF_PATH` environment variables.
 
+Tinychain's tensor feature can be installed using cargo using the command below.
+
+`cargo install tinychain --features=tensor`
+
+**Note** 
+Installing Tinychain feature requires to create an `arrayfire.pc` file for pkg-config. Create `arrayfire.pc` file in any folder of your choosing. Set the env variable `PKG_CONFIG_PATH` and point it to the folder you created the `arrayfire.pc` file in.
```

Comment:
this should be the tensor feature

Reply:
nit: this sentence doesn't need to end in a preposition

Reply:
Arghh!! Sorry I'll fix this. Late night edits 🤦🏾